src/FMBot.Bot/SlashCommands/TopSlashCommands.cs

```diff
@@ -52,16 +55,21 @@ public class TopSlashCommands : InteractionModuleBase
         [Summary("Billboard", "Show top artists billboard-style")] bool billboard = false,
         [Summary("User", "The user to show (defaults to self)")] string user = null,
         [Summary("XXL", "Show extra top artists")] bool extraLarge = false,
-        [Summary("Private", "Only show response to you")] bool privateResponse = false)
+        [Summary("Private", "Only show response to you")] bool privateResponse = false,
+        [Summary("Discogs", "Show top artists in Discord collection")] bool discogs = false)
```

Comment:
this says `Discord` instead of `Discogs`

Reply:
freudian slip, if one can call it that :sweat_smile: